examples/complete/main.tf

```diff
@@ -88,3 +88,15 @@ module "cis_glb" {
     }
   ]
 }
+
+##############################################################################
+# Enables web application firewall(WAF) to CIS instance
+##############################################################################
+
+module "cis_domain_settings" {
+  source          = "../../modules/waf"
+  cis_instance_id = module.cis_instance.cis_instance_id
+  domain_id       = module.cis_instance.cis_domain.domain_id
+  waf             = "on"
+  min_tls_version = "1.2"
```

Comment:
If it is set in sub-module, then no need to provide any value from example.

Reply:
Done